src/GovUk.Education.ExploreEducationStatistics.Content.Model/ReleaseStatus.cs

```diff
@@ -4,6 +4,6 @@ public enum ReleaseStatus
     {
         Draft,
         HigherLevelReview,
-        Live,
```

Comment:
Think we might still want the Live status?

Reply:
I suspect we will too - I will reintroduce it when we get to that piece of functionality